scripts/test/unit-cov.sh

```diff
@@ -0,0 +1,8 @@
+pytest \
+    -vv \
+    --cov=dagfactory \
+    --cov-report=term-missing \
+    --cov-report=xml \
+    --durations=0 \
+    -m "not (integration or perf)" \
+    --ignore=tests/test_example_dags.py
```

Comment:
```suggestion
    --durations=0
```
We do not have integration or perf ATM